test/integration/resolvers/lens.test.ts

```diff
@@ -8,15 +8,8 @@ describe('resolvers', () => {
       expect(result).toBe(false);
     });
 
-    it('should resolve with handle', async () => {
-      const result = await resolvers.lens('fabien.lens');
-
-      expect(result).toBeInstanceOf(Buffer);
-      expect(result.length).toBeGreaterThan(1000);
-    });
-
     it('should resolve with address', async () => {
-      const result = await resolvers.lens('0xeF8305E140ac520225DAf050e2f71d5fBcC543e7');
+      const result = await resolvers.farcaster('0x09CEdb7bb69f9F6DF646dBa107D2bAACda93D6C9')
```

Comment:
Can you remove change on this file?

Reply:
describe('resolvers', () => {
  describe('lens', () => {
    it('should return false if missing', async () => {
      const result = await resolvers.lens('0x556B14CbdA79A36dC33FcD461a04A5BCb5dC2A70');

      expect(result).toBe(false);
    });

    it('should resolve with handle', async () => {
      const result = await resolvers.lens('fabien.lens');

      expect(result).toBeInstanceOf(Buffer);
      expect(result.length).toBeGreaterThan(1000);
    });

    it('should resolve with address', async () => {
      const result = await resolvers.lens('0xeF8305E140ac520225DAf050e2f71d5fBcC543e7');

      expect(result).toBeInstanceOf(Buffer);
      expect(result.length).toBeGreaterThan(1000);
    });
  });
});

Reply:
fixed the copy & paste overwrite, sorry